roles/telemetry_autoscaling/tasks/creating_stack.yml

```diff
@@ -60,7 +60,7 @@
   until: '"CREATE_COMPLETE" in result.stdout'
     #  failed_when: '"CREATE_COMPLETE" not in result.stdout'
 
-- name: Verify that ceilometer_cpu metric exist
+- name: RHOSO-12707 Verify that ceilometer_cpu metric exist
```

Comment:
I think the test here is invalid as well.
It doesn't confirm that the ceilometer_cpu metric exists, it just confirms that the openstack metric list command actually succeeds.

I need to follow-up on this